base.css

```diff
@@ -1,103 +1,89 @@
-:root{
-    --background-color:#222831 ;
-    --darkbackground-color:#171c22;
+:root {
+  --background-color: #222831;
+  --darkbackground-color: #171c22;
 }
 
-
-body{
-    font-family: 'Quicksand', sans-serif;
-    background-color: var(--darkbackground-color);
-    color: var(--white-color);
+body {
+  font-family: "Quicksand", sans-serif;
+  background-color: var(--darkbackground-color);
+  color: var(--white-color);
 }
 
-.postion-absolute{
-    position: absolute;
+.postion-absolute {
```

Comment:
just a suggestion you can write pos-abs instead of writing whole position-absolute. using smaller names would reduce the clutter in you html class of the element.

Reply:
thanks got a spelling mistake